question.php

```diff
@@ -140,6 +140,31 @@ public function apply_attempt_state(question_attempt_step $step) {
         $this->step = $step;
     }
 
+    /**
+     * Get the spellchecking response.
+     *
+     * @param array $response
+     * @return string
+     * @throws coding_exception
+     * @throws dml_exception
+     * @throws moodle_exception
+     */
+    private function get_spellchecking(array $response):string {
+        $fullaiprompt = $this->build_full_ai_spellchecking_prompt($response['answer']);
+        $ai = new local_ai_manager\manager('feedback');
```

Comment:
This will fall over when using my back end because local_ai_manager does not exist. I have played with putting in a check to see if the class exists. However with the Moodle 4.5 AI subsystem due soon, I would like a way of checking for either my system, the mebis system, or the core system. Best wishes
Marcus